### PR TITLE
Remove idempotency token for Athena queries

### PR DESCRIPTION
**Pull request by fiskus**
# Description

# TODO

- [ ] Unit tests
- [ ] Automated tests (e.g. Preflight)
- [ ] Documentation
    - [ ] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
    - [ ] JavaScript: basic explanation and screenshot of new features
- [ ] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

**Review by nl0 — APPROVED** [review]
why do we need this change? is idempotency useless in this case?

**Comment by fiskus** [comment]
Documentation is a little confusing: it says that two requests with the same token will return the same query. It doesn't say about results. So, probably it means two simultaneously executed queries will execute only one query if they share the same token. But it seems it's already done internally by SDK, and this token is not required (they articulate it in docs, that it's not required)
And preserving this token in code can lead to bugs, and it's just unnecessary to maintain something we don't need.
---
TLDR:
1. It's not required
2. It can produce errors and additional work to maintain
